src/nbl/ui/CWindowWin32.cpp

```diff
@@ -237,18 +237,33 @@ namespace ui
 						event.type = SMouseEvent::EET_MOVEMENT;
 						event.movementEvent.movementX = rawMouse.lLastX;
 						event.movementEvent.movementY = rawMouse.lLastY;
+						event.movementEvent.isRelative = true;
 						event.window = window;
 						auto lk = inputChannel->lockBackgroundBuffer();
 						inputChannel->pushIntoBackground(std::move(event));
```

Comment:
this looks common to all if-satements, can you not push it outside?

Reply:
nah i don't think it's a good idea, cause i only want to push the events i need. it's a simple validation that stuff like absolute mouse movement won't get pushed

Reply:
ok